apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/ItemController.java

```diff
@@ -60,6 +60,10 @@ public OpenItemDTO createItem(@PathVariable String appId, @PathVariable String e
       throw new BadRequestException("User " + item.getDataChangeCreatedBy() + " doesn't exist!");
     }
 
+    if( (StringUtils.isEmpty(item.getComment()) ? 0 : item.getComment().length())  > 64){
```

Comment:
how about 

```java
if (!StringUtils.isEmpty(item.getComment()) && item.getComment().length() > 64) {
...
}
```

so that it may look simpler?

Reply:
yeah